docs/resources/azapi_resource_action.md

```diff
@@ -69,6 +69,28 @@ resource "azapi_resource_action" "stop" {
 }
 ```
 
+Here's an example to use `azapi_resource_action` resource to perform a provider action.
+
+```hcl
+terraform {
+  required_providers {
+    azapi = {
+      source = "Azure/azapi"
+    }
+  }
+}
+
+data "azapi_resource_action" "test" {
+  type        = "Microsoft.Resources/providers@2020-04-01-preview"
```

Comment:
Shouldn't the `type` here be `Microsoft.ResourceGraph@2020-04-01-preview`? The `Microsoft.Resources/providers` has no API version `2020-04-01-preview`, which seems only map to a small set of operations that are defined in the `resources` RP, e.g. https://github.com/Azure/azure-rest-api-specs/blob/e680f9bf4f154ec427ba7feac432ed8efd9665d0/specification/resources/resource-manager/Microsoft.Resources/stable/2022-09-01/resources.json#L2131.

On the other hand, the matching resource type and its API version is essential for the language server?